src/WebJobs.Script.WebHost/WebJobsScriptHostService.cs

```diff
@@ -269,7 +284,15 @@ private IHost BuildHost(bool skipHostStartup = false, bool skipHostJsonConfigura
 
         private ILogger GetHostLogger()
         {
-            var hostLoggerFactory = _host?.Services.GetService<ILoggerFactory>();
+            ILoggerFactory hostLoggerFactory = null;
+            try
+            {
+                hostLoggerFactory = _host?.Services.GetService<ILoggerFactory>();
+            }
+            catch (DryIoc.ContainerException)
```

Comment:
Ideally we'd avoid leaking the container details into the rest of the code. Is there another exception thrown? We can change this if needed.

Reply:
Reverted this change